samples/snippets/cpp/VS_Snippets_Remoting/NclSslServerSync/CPP/NclSslServerSync.cpp

```diff
@@ -53,8 +53,8 @@ public ref class SslTcpServer sealed
       // Authenticate the server but don't require the client to authenticate.
       try
       {
-         sslStream->AuthenticateAsServer( serverCertificate, false, 
-             SslProtocols::Tls, true );
+         sslStream->AuthenticateAsServer( serverCertificate, false, true );
```

Comment:
We should use named parameters (similar to the change in the other snippet below). Doing so means we can remove the comment as well.

Reply:
@davidsh I thought C++/CLI doesn't support named parameters.

Reply:
It's my understanding that C++ doesn't support named parameters.

Reply:
Yes, you're right.  I didn't realize from reading the markdown that it as describing C++ usage.  So, ignore my comments on that.